src/test/java/uk/gov/hmcts/darts/common/service/impl/DataAnonymisationServiceImplTest.java

```diff
@@ -273,22 +308,23 @@ void positiveAnonymiseTranscriptionEntity() {
         TranscriptionWorkflowEntity transcriptionWorkflowEntity2 = mock(TranscriptionWorkflowEntity.class);
         transcriptionEntity.setTranscriptionWorkflowEntities(List.of(transcriptionWorkflowEntity1, transcriptionWorkflowEntity2));
 
-        doNothing().when(dataAnonymisationService).anonymiseTranscriptionCommentEntity(any(), any());
+        doNothing().when(dataAnonymisationService).anonymiseTranscriptionCommentEntity(any(), any(), anyBoolean());
         doNothing().when(dataAnonymisationService).anonymiseTranscriptionWorkflowEntity(any());
 
 
         UserAccountEntity userAccount = new UserAccountEntity();
-        dataAnonymisationService.anonymiseTranscriptionEntity(userAccount, transcriptionEntity);
+        dataAnonymisationService.anonymiseTranscriptionEntity(userAccount, transcriptionEntity, isManuallyRequested);
 
-        verify(dataAnonymisationService, times(1)).anonymiseTranscriptionCommentEntity(userAccount, transcriptionCommentEntity1);
-        verify(dataAnonymisationService, times(1)).anonymiseTranscriptionCommentEntity(userAccount, transcriptionCommentEntity2);
+        verify(dataAnonymisationService).anonymiseTranscriptionCommentEntity(userAccount, transcriptionCommentEntity1, isManuallyRequested);
+        verify(dataAnonymisationService).anonymiseTranscriptionCommentEntity(userAccount, transcriptionCommentEntity2, isManuallyRequested);
 
-        verify(dataAnonymisationService, times(1)).anonymiseTranscriptionWorkflowEntity(transcriptionWorkflowEntity1);
-        verify(dataAnonymisationService, times(1)).anonymiseTranscriptionWorkflowEntity(transcriptionWorkflowEntity2);
+        verify(dataAnonymisationService).anonymiseTranscriptionWorkflowEntity(transcriptionWorkflowEntity1);
+        verify(dataAnonymisationService).anonymiseTranscriptionWorkflowEntity(transcriptionWorkflowEntity2);
     }
 
-    @Test
-    void positiveAnonymiseHearingEntity() {
+    @ParameterizedTest(name = "Anonymise hearing with isManuallyRequested = {0}")
+    @ValueSource(booleans = {true, false})
+    void anonymiseHearingEntity(boolean isManuallyRequested) {
```

Comment:
Can you amend the name to use the new convention please

Reply:
Updated